river/config.go

```diff
@@ -42,6 +42,11 @@ type Config struct {
 	FlushBulkTime TomlDuration `toml:"flush_bulk_time"`
 
 	SkipNoPkTable bool `toml:"skip_no_pk_table"`
+
+	ToLowerColumns bool `toml:"to_lower_fields"`
+
+	// It contains converted table(eg, lower case fields)
+	ConvertedTable []string
```

Comment:
1. If we have too many tables, the slice comparison will be very slow.
2. How do you handle DDL (we will support it later)

I don't think we need this field here, now the easiest way is to covert the field name every time (you may abstract a function to do this) when you set the key to the request data map in makeInsertReqData and makeUpdateReqData.